tools/rdm/TestDefinitions.py

```diff
@@ -777,6 +788,24 @@ def VerifyResult(self, response, fields):
         self.AddAdvisory('%s supported but %s is not' %
                          (pid_names[0], ','.join(unsupported_pids)))
 
+    for pid_names in self.PID_REVERSE_DEPENDENCIES:
+      if self.LookupPid(pid_names[0]).value in supported_parameters:
+        continue
+
+      dependant_pids = []
+      for pid_name in pid_names[1:]:
+        pid = self.LookupPid(pid_name)
+        if pid is None:
+          self.SetBroken('Missing PID %s' % pid_name)
```

Comment:
Hmm, this seems wrong. Broken is supposed to indicate a code error. I think this (and the one above) should be SetFailed() 

See https://www.openlighting.org/rdm-tools/rdm-responder-tests/interpreting-the-test-output/